doc/OnlineDocs/citing_pyomo.rst

```diff
@@ -8,6 +8,8 @@ Hart, William E., Jean-Paul Watson, and David L. Woodruff. "Pyomo: modeling and
 
 Hart, William E., Carl Laird, Jean-Paul Watson, David L. Woodruff, Gabriel A. Hackebeil, Bethany L. Nicholson, and John D. Siirola. Pyomo – Optimization Modeling in Python. Springer, 2017.
 
+Bynum, Michael L., Gabriel A. Hackebeil, William E. Hart, Carl D. Laird, Bethany L. Nicholson, John D. Siirola, Jean-Paul Watson, and David L. Woodruff. Pyomo - Pptimization Modeling in Python. Springer, 2021.
```

Comment:
You should also remove the  reference to the 2nd edition.

Reply:
It has the first and second in this file - do you want to go only to the 3rd?

Reply:
It doesn't - it has the original journal article and the second edition of the book.  We should only have the most recent edition (and at this point, we should reverse the order of the references and list the book first)